app/concerns/bulk_updates.rb

```diff
@@ -12,6 +12,36 @@ def retrieve_bulk_updates
                      []
                    end
 
-    bulk_updates
+    bulk_updates.each { |update| hydrate_task(update) }
+
+    bulk_updates.sort_by { |option| option['task-id'] }
+  end
+
+  # When the bulk update task comes back from CMR it needs a bit of
+  # massaging before its ready for the view
+  def hydrate_task(task, flatten_keys: true)
+    task['request-json-body'] = JSON.parse(task['request-json-body'])
+
+    if flatten_keys
+      # Pull out all of the keys from the originial
+      # request object and insert them into the root
+      task['request-json-body'].each do |key, value|
+        task[key] = value
+      end
+    end
+  end
+
+  # The response from CMR only includes the concept-id
+  # but for display purposes we need detailed information
+  def hydrate_collections(bulk_update)
```

Comment:
very minor thing, but you're passing `@task` into both the hydrate methods, but in `hydrate_task` you call the variable `task` and for `hydrate_collections` you call it `bulk_update`. I know we're using them interchangeably but seems odd since the two methods are so close together (both in the concern and in usage).